jax/_src/numpy/lax_numpy.py

```diff
@@ -3746,6 +3746,9 @@ def array_equal(a1: ArrayLike, a2: ArrayLike, equal_nan: bool = False) -> Array:
 
   JAX implementation of :func:`numpy.array_equal`.
 
+  This function will return ``False`` if the input arrays cannot be broadcasted
+  to the same shape.
+
```

Comment:
I think this ended up in the wrong spot?

Reply:
ops, its fixed.